src/In.ProjectEKA.HipService/DataFlow/CryptoHelper/CryptoHelper.cs

```diff
@@ -0,0 +1,118 @@
+namespace In.ProjectEKA.HipService.DataFlow.CryptoHelper
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Security.Cryptography;
+    using Logger;
+    using Newtonsoft.Json;
+    using Org.BouncyCastle.Crypto;
+    using Org.BouncyCastle.Crypto.Digests;
+    using Org.BouncyCastle.Crypto.EC;
+    using Org.BouncyCastle.Crypto.Generators;
+    using Org.BouncyCastle.Crypto.Parameters;
+    using Org.BouncyCastle.Security;
+    using Encoder = Org.BouncyCastle.Utilities.Encoders;
+
+    public class CryptoHelper: ICryptoHelper
```

Comment:
Why we need an interface? And don't call it as Helper.

Reply:
Will change the name. Interface is required for testing dataflow client